doc/module2rst.py

```diff
@@ -60,9 +43,14 @@ def gen_rst(module, f):
 
 """)
 
-    cs = children(module, recursive=True)
-    for c in cs:
-        cname = c.__name__
+    for cpath in glob(module_path + "/**/*.py", recursive=True):
+        if not os.path.exists(cpath):
+            continue
+        if "__pycache__" in cpath:
+            continue
+        if "__init__.py" in cpath:
```

Comment:
@kan-bayashi I misunderstood your question. I explicitly removed the `__init__.py`. I will revert that.